ducktape/command_line/main.py

```diff
@@ -181,7 +182,12 @@ def main():
         (cluster_mod_name, cluster_class_name) = args_dict["cluster"].rsplit('.', 1)
         cluster_mod = importlib.import_module(cluster_mod_name)
         cluster_class = getattr(cluster_mod, cluster_class_name)
-        cluster = cluster_class(cluster_file=args_dict["cluster_file"])
+
+        cluster_kwargs = {"cluster_file": args_dict["cluster_file"]}
+        checkers = [load_function(func_path) for func_path in args_dict["ssh_checker_function"]]
```

Comment:
There needs to be a readable error message if the function cannot be loaded for some reason.

Reply:
added to the load function method